stumpy/core.py

```diff
@@ -4371,7 +4371,7 @@ def get_ray_nworkers(ray_client):
 
 
 @njit
-def _update_egress_PI(D, P, I, excl_zone, n_appended=0):
+def _update_incremental_PI(D, P, I, excl_zone, n_appended=0):
```

Comment:
Why change the name of the function if the function itself has not changed??

Reply:
> Why change the name of the function if the function itself has not changed??

I am calling this function from both methods `_update` and `_update_egress` of `stumpy.stumpi` / `stumpy.aampi`. So, I thought using the term `_egress` might be misleading here. But, `_update_PI` is not clear enough. So, that is why I decided to go with `_update_incremental_PI`.

Reply:
Nevermind, I didn't realize it was a new function altogether. I only saw the diff between commits.